packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenu.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Options menu"
 cssPrefix: "pf-c-options-menu"
-propComponents: ['OptionsMenu', 'OptionsMenuItem', 'OptionsMenuSeparator', 'OptionsMenuToggle', 'OptionsMenuToggleWithText']
```

Comment:
Since this just got merged to master please readd `propComponents: ['OptionsMenu', 'OptionsMenuItem', 'OptionsMenuSeparator', 'OptionsMenuToggle', 'OptionsMenuToggleWithText']`